templates/zerver/emails/confirm_demo_organization_email.subject.txt

```diff
@@ -0,0 +1 @@
+{{ _("Verify email address for demo organization owner") }}
```

Comment:
I think we could just use `confirm_new_email_subject.txt` here without customizing.